### ISSUE TITLE
Udiskie fails to mount devices after upgrade

### ISSUE THREAD

**Issue by corpix** [NixOS/nixpkgs#30506]
## Issue description

I have upgraded udiskie to 1.7.1 on my system and now it fails to unlock the LUKS encrypted drives with error:
```
udiskie-start[1912]: failed to unlock /org/freedesktop/UDisks2/block_devices/sdc1: __call__() takes exactly 2 arguments (3 given)
```

I have a nixpkgs repository checked out, reverted #30363, now it works fine.

### Steps to reproduce

1. Install 1.7.1 version
2. Plug in LUKS encrypted flash drive
3. See a previously mentioned error in systemd journal

## Technical details

* System: 18.03.git.f6c10ad2e6 (Impala)
* Nix version: nix-env (Nix) 1.11.15
* Nixpkgs version: 18.03.git.f6c10ad2e6
* Sandboxing enabled: build-use-sandbox = true

**Comment by corpix** [comment]
Here is my udisks2 & udiskie configuration:

``` nix
{ config, pkgs, ... }:
let
  wrapSystemdScript = (
    program: ''
      source ${config.system.build.setEnvironment}
      exec ${program}
    ''
  );
in rec {
  environment.systemPackages = with pkgs; [
    udiskie
    udisks2
    lxqt.lxqt-openssh-askpass
  ];
  services = {
    udisks2.enable  = true;
  };
  systemd.user.services = {
    udiskie = {
      description = "Mounts disks in userspace with udisks";
      serviceConfig = {
        Type    = "simple";
        Restart = "always";
      };
      script = wrapSystemdScript ''
        ${pkgs.udiskie}/bin/udiskie \
          -2                        \
          --smart-tray              \
          --automount               \
          --no-config               \
          --no-password-cache       \
          --no-notify               \
          --password-prompt ${pkgs.lxqt.lxqt-openssh-askpass}/bin/lxqt-openssh-askpass
      '';
      wantedBy = [ "default.target" ];
    };
  };
  users.extraUsers.user = {
    extraGroups = [
      "plugdev"
    ];
  };
  users.extraGroups.plugdev = { };
}
```

**Comment by Moredread** [comment]
Mhh, that could be an upstream problem...

**Comment by corpix** [comment]
Looks like it was fixed in #30526, so closing this.
